cardano-chain-gen/test/testfiles/config-conway/test-db-sync-config-bootstrap.json

```diff
@@ -0,0 +1,117 @@
+{
+  "EnableLogMetrics": false,
+  "EnableLogging": true,
+  "NetworkName": "testing",
+  "NodeConfigFile": "test-config.json",
+  "PrometheusPort": 8080,
+  "RequiresNetworkMagic": "RequiresMagic",
+  "defaultBackends": [
+    "KatipBK"
+  ],
+  "defaultScribes": [
+    [
+      "StdoutSK",
+      "stdout"
+    ]
+  ],
+  "minSeverity": "Info",
+  "options": {
+    "cfokey": {
+      "value": "Release-1.0.0"
+    },
+    "mapBackends": {},
+    "mapSeverity": {
+      "db-sync-node": "Info",
+      "db-sync-node.Mux": "Error",
+      "db-sync-node.Subscription": "Error"
+    },
+    "mapSubtrace": {
+      "#ekgview": {
+        "contents": [
+          [
+            {
+              "contents": "cardano.epoch-validation.benchmark",
+              "tag": "Contains"
+            },
+            [
+              {
+                "contents": ".monoclock.basic.",
+                "tag": "Contains"
+              }
+            ]
+          ],
+          [
+            {
+              "contents": "cardano.epoch-validation.benchmark",
+              "tag": "Contains"
+            },
+            [
+              {
+                "contents": "diff.RTS.cpuNs.timed.",
+                "tag": "Contains"
+              }
+            ]
+          ],
+          [
+            {
+              "contents": "#ekgview.#aggregation.cardano.epoch-validation.benchmark",
+              "tag": "StartsWith"
+            },
+            [
+              {
+                "contents": "diff.RTS.gcNum.timed.",
+                "tag": "Contains"
+              }
+            ]
+          ]
+        ],
+        "subtrace": "FilterTrace"
+      },
+      "#messagecounters.aggregation": {
+        "subtrace": "NoTrace"
+      },
+      "#messagecounters.ekgview": {
+        "subtrace": "NoTrace"
+      },
+      "#messagecounters.katip": {
+        "subtrace": "NoTrace"
+      },
+      "#messagecounters.monitoring": {
+        "subtrace": "NoTrace"
+      },
+      "#messagecounters.switchboard": {
+        "subtrace": "NoTrace"
+      },
+      "benchmark": {
+        "contents": [
+          "GhcRtsStats",
+          "MonotonicClock"
+        ],
+        "subtrace": "ObservableTrace"
+      },
+      "cardano.epoch-validation.utxo-stats": {
+        "subtrace": "NoTrace"
+      }
+    }
+  },
+  "rotation": {
+    "rpKeepFilesNum": 10,
+    "rpLogLimitBytes": 5000000,
+    "rpMaxAgeHours": 24
+  },
+  "setupBackends": [
+    "AggregationBK",
+    "KatipBK"
+  ],
+  "setupScribes": [
+    {
+      "scFormat": "ScText",
+      "scKind": "StdoutSK",
+      "scName": "stdout",
+      "scRotation": null
+    }
+  ],
+  "insert_options": {
+    "tx_out": "bootstrap"
+  }
+}
```

Comment:
unsure if you are already doing this but could these not be a single file with updated value, here is a simple example:
https://github.com/IntersectMBO/cardano-db-sync/blob/045330da2352504f67f68a0b870cc5e08951810b/cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Conway/CommandLineArg/MigrateConsumedPruneTxOut.hs#L60-L65

but for this scenarios it would be: 
```haskell
  where
    mkSynNodeConfig :: IO SyncNodeConfig
    mkSynNodeConfig = do
      -- have 1 init file that is read and parsed
      initConfigFile <- mkSyncNodeConfig conwayConfigDir
      -- update the results with required values for the specific test in above tests 
      -- it's "tx_out": "bootstrap" so that is reflected as so
      pure $ 
        initConfigFile 
          { dncInsertConfig = initInsertConfig { spcTxOut = TxOutBootstrap } }

-- have an init for specific set of tests or all tests if required 
initInsertConfig :: SyncInsertConfig
initInsertConfig = 
  SyncInsertConfig
    { spcTxOut = TxOutEnable
    , spcLedger = LedgerEnable
    , spcShelley = ShelleyDisable
    ...
  }
```

Reply:
I've written a couple new tests in this style, but the majority I've kept unchanged